README.md

```diff
@@ -162,13 +163,17 @@ Verification inputs:
                         used with multiple inputs (default: None)
   --signature FILE      The signature to verify against; not used with
                         multiple inputs (default: None)
+  --rekor-bundle FILE   The offline Rekor bundle to verify with; not used with
+                        multiple inputs (default: None)
 
 Extended verification options:
   --cert-email EMAIL    The email address to check for in the certificate's
                         Subject Alternative Name (default: None)
   --cert-oidc-issuer URL
                         The OIDC issuer URL to check for in the certificate's
                         OIDC issuer extension (default: None)
+  --rekor-offline       Perform offline Rekor verification using a bundle;
+                        implied by --bundle (default: False)
```

Comment:
```suggestion
                        implied by --rekor-bundle (default: False)
```
Right? TBH, I'm slightly concerned about too much rekor-specific stuff sneaking into the UI, especially for something that a user might want to use (bundles) that doesn't require knowledge of rekor specifically. Maybe this should just be `--bundle`?

Reply:
I tend to agree, although I think @haydentherapper's point was that `--bundle` will eventually be the expected CLI option for the "sigstore bundle", whereas we're currently consuming the (confusingly named) "rekor-only bundle". So having it as `--rekor-bundle` will make the migration a little smoother for users, as they won't need to change their inputs for the same flag between sigstore-python versions.

That being said, maybe I could add a warning log when this flag is used, to emphasize to users that it'll eventually (soon?) be replaced by `--bundle`?